internal/netutil/ident.go

```diff
@@ -162,25 +161,25 @@ func uidFromLsof(lip net.IP, lport int, rip net.IP, rport int) (uid int, err err
 		//"-a",  // AND the following together:
 		// "-u", strconv.Itoa(uid)  // just this uid
 		"-itcp") // we only care about TCP connections
+
 	stdout, err := cmd.StdoutPipe()
 	if err != nil {
-		return
+		return 0, err
 	}
-	defer cmd.Wait()
```

Comment:
I'd do something like:

```
defer func() {
   if werr := cmd.Wait(); werr != nil && err == nil {
        err = werr
   }
}()
```

... otherwise cmd.Wait in your proposed changed takes priority over any error from `uidFromUsername` which is more interesting.